### ISSUE TITLE
The backdrop-filter applied to the UI elements resulted in changes to the entire VSCode text UI.

### ISSUE THREAD

**Issue by markox92** [microsoft/vscode#204788]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version: 1.86.1
- OS Version: Windows 11 Pro (22631.3085) & Ubuntu 23.10 (Gnome 45,44)

Steps to Reproduce:

1. Toggle Developer Tools and play with backdrop filters.
2. Added backdrop filters (blur, opacity, saturation... or any other) rule to context menu, widgets, or any other elements. The whole VSCode UI was changed, but only the text... this is very strange.
3. No extension installed, only one theme, but also I have try with default theme from VScode and same things is happened.

Here is an animated GIF where the backdrop is enabled and disabled in context menu; you can see what happened. Try to zoom and check the letters carefully

![ezgif-7-8a72302542](https://github.com/microsoft/vscode/assets/159203552/fae48953-67a3-4974-a254-889c4efe6c1e)

Also, here is a zoomed gif of the some text
![1165cfced7](https://github.com/microsoft/vscode/assets/159203552/17e4850b-a0c7-48b8-835e-c6bebd44ff43)

Why has the text in the editor and UI changed if I just added a backdrop-filter to the context menu, for example?

**Comment by deepak1556** [comment]
I am unable to repro with 1.96, please confirm if the issue persists with latest stable

**Comment by markox92** [comment]
> I am unable to repro with 1.96, please confirm if the issue persists with latest stable

Yes, the problem is still there, sorry about the quality, this is how snipping tools record videos, but anyway, you can see the fonts and the problem if you watch this in fullscreen.

https://github.com/user-attachments/assets/93821dae-aaac-4e09-b91b-e4116e15c02d
